doc/reliability.rst

```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
+updates, building/launching statistics, and reliability metrics.
+
+Site Reliability Goals
```

Comment:
I would add a section: "Expectations for mybinder.org users"
Let's use some of Fernando's wording

---
*ABSOLUTELY*  This is, for now, purely a best-effort, experimental research project. We're *not* staffed 24/7, nor do we have the budget, capacity or intent at this point to go there.  

Communicating this clearly on the homepage could help: to say that this is a *research pilot*, whose main goal is to understand usage patterns and workloads for future evolution, but *not* a service that anyone can rely on for anything critical, would go a long way.

Reply:
Where is this text from? :)

Reply:
From @fperez's wonderful mind @yuvipanda 👍